modules/ROOT/pages/datamapper-fixed-width-input-format.adoc

```diff
@@ -218,7 +218,7 @@ image::5-field-attr.png[5.Field.attr]
 
 . We will now create the additional fields. In the *Input* pane, right-click the top-level element (in this case, `fwtest`), then select *Add Field*.
 +
-image::6-add-field.png[6.add_field]
+image::6-add-field.png["A blue button with a plus sign and the text "Add Field""]
```

Comment:
```suggestion
image::6-add-field.png["A blue button with a plus sign and the text Add Field"]
```

Reply:
I think it's okay to forgo the quotes and other formatting in alt text. I'm not sure if the "" would have messed up anything. Maybe not. But just to be safe.